packages/logger/src/index.ts

```diff
@@ -45,15 +45,26 @@ export const logger: ILogger = {
 
 // tslint:disable-next-line: variable-name
 export const NSlogger = (namespace: string = ''): ILogger => {
+  let namespaceModified = '';
+  const service = instance.defaultMeta.serviceContext.service;
+
+  if (service && namespace) {
+    namespaceModified = `${service}:${namespace}`;
+  } else if (service && !namespace) {
+    namespaceModified = service;
+  } else {
+    namespaceModified = namespace;
+  }
+
```

Comment:
Move this into a separate function called `getNamespace` please. That way you don't need to use a `let` and also adds less functionality into the main logger function